src/ui/src/partials/admin_garden_index.html

```diff
@@ -1,14 +1,24 @@
 <h1 class='page-header'>
   <span>Garden Management</span>
   <span class="pull-right" ng-show="responseState(response) !== 'loading'">
+    <button type="button" class="btn btn-primary" ng-click="createGardenFormHide = false">Create Garden</button>
     <button type="button"
             class="btn btn-primary"
             ng-click="syncGardens()">Sync All</button>
   </span>
 </h1>
-  
+<div class="pull-right">
+  <div class="form-popup" id="myForm" ng-hide=createGardenFormHide>
+    <label><b>Garden Name</b></label>
+    <input type="text" placeholder="Enter Garden Name" name="garden_name" required ng-model="garden_name">
+
+    <button type="button" class="btn" ng-click="createGarden()">Create</button>
+    <button type="button" class="btn" ng-click="createGardenFormHide = true">Close</button>
```

Comment:
On close, we should clear the value being set by the form. That way the user gets a clean form if they reopen it.